Voucherify Source catalog docs update. (#4796)
segmentio/segment-docs: src/connections/sources/catalog/cloud-apps/voucherify/index.md

Voucherify uses the stream Source component to send Segment event data. It uses a server-side `track` method to send data to Segment. These events are then available in any destination that accepts server-side events, and available in a schema in your data warehouse, so you can query using SQL. These events are also assigned to user accounts in the Segment Profiles (Unify). 

> The default behavior is for Voucherify to pass the `userId` associated with the customer. Events sent to the Voucherify source are only the events that the customer triggered, so there is no situation where `anonymousId` is sent. The Voucherify Source will not send anonymous events to Segment. This means that events that are not assigned to customer and not called by the customer will be ignored. The `source_id` field from Voucherify will become the `userId` field in Segment.

Reviewer: ```suggestion
By default, Voucherify passes a customer's associated `userId` to Segment. Because Voucherify sends customer-triggered events only, `anonymousId` is not used. The Voucherify source doesn't send anonymous events to Segment. As a result, events that aren't assigned to or called by a customer are ignored. For mapping, `source_id` on Voucherify events becomes the `user_id` in Segment.
```

Author: Sure, I'll commit right now

Author: sure

Author: is everything OK now?
